opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/accounts/AccountListingService.java

```diff
@@ -31,6 +33,7 @@ protected void doValidate(DelegateExecution execution, Xs2aContext context) {
 
     @Override
     protected void doRealExecution(DelegateExecution execution, Xs2aContext context) {
+        log.debug("do real for:{}", context.toString());
```

Comment:
I would suggest removing this. If we need logging it should be either handled by Aspect or by ValidatedExecution parent class. With flowable - you can simply use (in case of disabled encryption) 
act_ru_execution, act_ru_variable, act_hi_execution, act_hi_variable tables

Reply:
done